src/deck/installation.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Installation
+---
+
+decK is entirely written in Go. The build process builds a single static binary,
+which makes it easy and convenient to install decK.
+
+## Prerequisites
+You have installed {{site.ce_product_name}} >= 1.x or
+{{site.ee_product_name}} >= 0.35.
+
+You can follow along installation instructions based on your
+Operating System (OS):
+
+## macOS
+
+If you are on macOS, install decK using brew:
+
+```shell
+$ brew tap kong/deck
+$ brew install deck
+```
+
+## Linux
+
+If you are Linux, you can either use the Debian or RPM archive from
+the Github [release page](https://github.com/kong/deck/releases)
+or install by downloading a compressed archive, which contains the binary:
+
+{% if_version eq:1.12.x %}
+```shell
+$ curl -sL https://github.com/kong/deck/releases/download/v1.12.0/deck_1.12.0_linux_amd64.tar.gz -o deck.tar.gz
+$ tar -xf deck.tar.gz -C /tmp
+$ sudo cp /tmp/deck /usr/local/bin/
+```
+{% endif_version %}
```

Comment:
The only issue I ran into is this one: this file is going to be shared, but i don't know what variable to use to refer to the current real version of the product (ie, not 1.12.x but 1.12.0). In previous versions we use this:

```
{{page.kong_versions[5].version}}
```
This obviously won't work, since the index needs to be bumped manually, so it would only ever show the current version. Is there a way we can pull the release number (1.12.0)? We have a plugin generating the 1.12.x version variable; should we also have it generate the release number (1.12.0) variable?

Reply:
Once #3853 is merged you can rebase this PR and reference `page.version` and `page.release` as needed

You'll also need to rename `version` to `release` in the `app/_data/docs_nav_deck_1.12.xyml` file